src/examples/printInterfaces.ts

```diff
@@ -14,16 +14,37 @@ export interface Bar {
 export interface Baz {
     baz: string;
 }
+
+export interface WithMethods {
+    myMethod(a: string): number;
+}
 `;
 
 const ast = createFromString(declarationFile);
 
-const interfaces: { name: string; members: string[] }[] = [];
+const interfaces: { name: string; properties: string[]; methods: string[] }[] = [];
 accept(ast, {
   [ts.SyntaxKind.InterfaceDeclaration]: (interfaceDeclaration) => {
+    const properties: string[] = [];
```

Comment:
- `interfaceDeclaration` is of type `InterfaceDeclarationWithTypeNarrowing`. See `src/accept/types.ts`.
- `interfaceDeclaration.members` is of type `(ts.PropertySignature | ts.MethodSignature)[]`.
- `member` is thus of type `(ts.PropertySignature | ts.MethodSignature)`.